src/components/PullRequestStatus.tsx

```diff
@@ -99,11 +116,35 @@ function getBadges(state: PullRequestState): JSX.Element[] {
   return badges;
 }
 
+function getCheckStatusBadge(checkStatus?: CheckStatus): JSX.Element[] {
+  switch (checkStatus) {
```

Comment:
Just confirming what will the value be when there are no checks (and no expected checks either)?

Reply:
w/ no checks, this will be `undefined`, hence the `checkStatus?`

Reply:
Indeed, sorry :)